sdm/src/main/java/com/sap/cds/sdm/service/SDMService.java

```diff
@@ -40,4 +40,11 @@ public void readDocument(
       SDMCredentials sdmCredentials,
       AttachmentReadEventContext context)
       throws IOException;
+
+  public int renameAttachments(
+      String jwtToken, SDMCredentials sdmCredentials, String fileName, String objectId)
```

Comment:
can we add fileName and objectId to cmisdocument and pass cmisdocument as parameter

Reply:
Done